connectors/tests/test_sync.py

```diff
@@ -512,12 +511,24 @@ async def test_connector_service_poll_large(mock_responses, patch_logger, set_en
     assert_re(".*Sync done: 1001 indexed, 0  deleted", patch_logger.logs)
 
 
+@pytest.mark.asyncio
+async def test_connector_service_poll_suspended_suspends_job(
```

Comment:
Hi @artem-shelkovnikov , curious about how you simulate a `SIGTERM` signal?

Reply:
Hi @wangch079, for this particular test I don't need it, for manual testing I do `kill -15 <pid>`.

Reply:
Is this test case testing the service can mark a running job as `suspended` when the service shuts down?

Reply:
Yup, `create_and_run_service(MEM_CONFIG_FILE, 0.1)` terminates the service in 0.1 second by calling `service.stop`, and this code cancels and marks jobs as suspended!